Lib/idlelib/query.py

```diff
@@ -220,6 +220,14 @@ def entry_ok(self):
             self.showerror("loader does not support get_filename",
                       parent=self)
```

Comment:
Delete existing errorneous argument.
```suggestion
            self.showerror("loader does not support get_filename".)
```
Previous two messages should have '.' added.